benchmarks/dc_eis_throughput/dceisThroughput.json

```diff
@@ -3,7 +3,7 @@
  "name": "node-dc-eis Ops/s[higher is better]" ,
  "template": "../common/common_template1.html.base",
  "units": "ops/s",
- "outputBase" : "dceis_throughput",
```

Comment:
same comment as above will change the name of the external graphs so the main page would also have to be updated.